ichnaea/log.py

```diff
@@ -25,29 +26,43 @@ def configure_logging():
     logging_level = settings("logging_level")
 
     if local_dev_env:
-        handlers = ["console"]
+        handlers = ["dev"]
```

Comment:
`dev` felt like a better name than `console` to me, since `mozlog` also outputs to the console.